### ISSUE TITLE
Import database fails using rest api

### ISSUE THREAD

**Issue by dehbmarques** [orientechnologies/orientdb#3431]
Hi,
When I try to import a database using rest, I get an exception:

```
java.lang.StringIndexOutOfBoundsException: String index out of range: -11
```

Exported file and the full response:
https://gist.github.com/dehbmarques/cae28225f041fe231903

OrientDB version: 2.0 Final

**Comment by lvca** [comment]
Yeah, this is an open issue. I suggest you to use console to import db.

**Comment by compensator** [comment]
@lvca why was this changed from a bug?  Is this fixed now?  We were considering using the REST interface for loads as well.

**Comment by lvca** [comment]
Sorry, I'm using an external tool to manage GitHub issues, and I realize that renaming things cause a drop of labels on GitHub. This is definitely a bug. I remember the HTTP Upload didn't work correctly. We're almost sure to replace current HTTP layer with Jetty, so this issue has low priority. Could you import the database through console?

**Comment by compensator** [comment]
We will forego the HTTP API and use the Graph API since we'll be migrating data from an RDBMS to an OrientDB Graph Model.  We had been considering the HTTP interface due to the strawman concept that we would translate the RDBMS star schema into JSON documents per table row, then insert each JSON document via the HTTP API with Gremlin as the language type, with each JSON document being a graph Vertex where Edges represent all primary-foreign key relationships in the RDBMS.

**Comment by josebalius** [comment]
Is there any progress on this?

**Comment by tglman** [comment]
Hi,

This should be already fixed in 2.2.x, closing 

Regards
